### ISSUE TITLE
[NetSDKE2E]Installing .NET 7.0 RC2 SDK, .NET Runtime version is RC1

### ISSUE THREAD

**Issue by vdanche** [dotnet/runtime#75105]
### Description

Installing .NET 7.0 RC2 SDK, .NET Runtime version is RC1

### Reproduction Steps

Download and install .NET 7 RC2 SDK (7.0.100-rc.2.22454.1) from Release/7.0.1xx" column in https://github.com/dotnet/installer

### Expected behavior

The .NET Runtime version should be 7.0.100-rc.2.xxx.

### Actual behavior

The .NET Runtime version is 7.0.0-rc.1.22422.12
![image](https://user-images.githubusercontent.com/65637796/188559525-078229d6-c357-443d-bc5b-c1b2e8c6c9f0.png)

**Comment by jkotas** [comment]
dotnet/runtime rc2 build was not merged into the installer repo yet. This will be fixed once https://github.com/dotnet/installer/pull/14412 is merged.

**Comment by vdanche** [comment]
Fixed on 7.0.100-rc.2.22466.28(runtime-7.0.0-rc.2.22464.1)
